eap/eap64-tx-recovery-s2i.json

```diff
@@ -0,0 +1,579 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "iconClass": "icon-eap",
+            "tags": "eap,javaee,java,jboss",
+            "version": "1.4.7",
+            "openshift.io/display-name": "JBoss EAP 6.4 (tx recovery)",
+            "openshift.io/provider-display-name": "Red Hat, Inc.",
+            "description": "An example EAP 6 application. For more information about using this template, see https://github.com/jboss-openshift/application-templates.",
+            "template.openshift.io/long-description": "This template defines resources needed to develop Red Hat Enterprise Application Server 6.4 based application, including a build configuration, application deployment configuration and insecure communication using http.  The template also demonstrates how to enable transaction recovery on scale down of application pods.",
+            "template.openshift.io/documentation-url": "https://access.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/",
+            "template.openshift.io/support-url": "https://access.redhat.com"
+        },
+        "name": "eap64-basic-s2i"
```

Comment:
@knrc Hey Kevin, please update template name here, s/eap64-basic-s2i/eap64-tx-recovery-s2i/ (without it attempt to deploy the template will overwrite the existing "eap64-basic-s2i" template instead of installing a new TX one).

Reply:
@iankko changes made, thanks for catching this